python/ops.py

```diff
@@ -83,37 +83,31 @@ def c2r_ifft2(x_fft):
   if ndims != 4:
     raise ValueError('Expecting ndims == 4, actual={}'.format(ndims))
   return c2r(
-      tf.transpose(
-          tf.signal.ifft2d(tf.transpose(x_fft, [0, 3, 1, 2])), [0, 2, 3, 1]))
+    tf.transpose(
+      tf.signal.ifft2d(tf.transpose(x_fft, [0, 3, 1, 2])), [0, 2, 3, 1]))
```

Comment:
Avoid reformatting the parts of the code that you're not touching, if you can. I can try to get a formatter up and running on github, I think there should be some facility for auto-formatting files according to the google python style guide.